Remnant: Cognizance 1 (#4871)
endless-sky/endless-sky: data/Remnant Cognizance.txt

mission "Remnant: Cognizance 1"
	name "Follow-up Observations"
	description "Plume would like you to do a follow up visit to the planets in <system>"
	minor
	source "Aventine"
	stopover "Nasqueron"
	stopover "Slylandro"
	cargo "scanning equipment" 8
	blocked "The Remnant researcher has another job for you, but you're going to have to free up some cargo space in order to take it on."
	to offer
		has "event: remnant: wait for plume"
		has "Remnant: Learn Sign Follow Up: declined"
		random < 50
	on offer
		conversation
			`A familiar face is waiting for you as you step into the crowded spaceport on Aventine. "Captain <first>!" signs Plume. "I have been hoping you would stop by." He gestures for you to sit down on the bench next to him.`

> 			`	"Now that most of the cocoons have been returned, I would like you return to Nasqueron and Slylandro to observe the void sprites' behavior. It is my hope that there might be a change that we can document for greater insights into their life cycles."`

Author: fixed. Thanks!

			choice
				`	"Sure, I could do that."`
					goto rc1accept
				`	"Not right now, but I could later."`
					defer
				`	"How much will I get paid?"`
			`	"Once the research is published, you will receive one million credits.`
			choice
				`	"I accept."`
				`	"Not right now, but I could later."`
					defer
			label rc1accept

> 			`	"Excellent. As before, please visit both Nasqueron and Slylandro, record the void sprites, and return here."`

Author: fixed.

			`	"Here, look!" He rewinds the recording and gestures curtly at the projection in front of you. "There. That one is injured. Some kind of burn in a long, straight line." He zips through the footage farther "and here, a scattering of scorch marks." He looks up at you and snaps out a curt series of gestures: "This is outside my experience, I need to consult an expert. Meet me in the spaceport in an hour."`

mission "Remnant: Cognizance 2"
	name "Eliminate Korath Ship"
	description "Plume would like you to eliminate the Korath ship in the Nenia system."
	source "Aventine"
	waypoint
		system "Nenia"
	to offer
		has "Remnant: Cognizance 1: done"
	on offer
		conversation
			`You arrive in the spaceport to find Plume pacing quickly along a walkway. He gestures a quick greeting and jumps straight to the point. "We have never seen marks like these naturally appear on void sprites, but they match the weapon scoring patterns we have seen delivered by Korath ships. As such, we believe the Korath are hunting the void sprites."`

> 			`	He looks at you with a surprising amount of fire. "I will not have our work ruined by reptilian warmongers. Destroy the Korath ship, then we can continue the research."`

Author: fixed.

				accept
	npc kill
		government "Korath"
		system "Nenia"
		personality forbearing timid uninterested surveillance staying
		ship "Korath Raider" "Korath Raider (Ember)"
			cargo "void sprite" 5 20
		dialog "You have destroyed the Korath Vessel. You can now return to <planet> to collect your payment."
	on complete
		event "korath hunting"
		conversation
			`When you return to Aventine, you find Plume waiting for you. He eagerly reviews the recording of the combat. "Thank you for eliminating this hunter..." he is interrupted by a flashing light on his commlink. He briefly reads something, then looks back up to you. "Apparently something about the Korath hunting the void sprites has concerned the defence prefect. Chilia has requested that you meet with him on Caelian immediately."`

event "korath hunting"
	government "Korath"
		"attitude toward"
			"Indigenous Lifeform" -.01
			"Void Sprite" -.01

mission "Remnant: Cognizance 3"
	name "Search for Korath"
	description "Prefect Chilia has asked you to track down a Korath fleet"
	source "Caelian"
	landing
	waypoint
		system "Aescolanus"
	to offer
		has "Remnant: Cognizance 2: done"
	on offer

> 		log "People" "Chilia" `Prefect Chilia is unusual compared to most military leaders you have met. He is inclined to do things himself, and rarely stands on protocol.`

Author: Fixed. Thanks!

		"remnant chilia" ++
		conversation
			branch chilia
				"remnant chilia" > 1

> 			`The <ship> has barely come to a rest on its designated pad when your panel is blinking with a request for permission to come aboard. A quick glance at the viewscreen shows a fit young man with a military bearing standing at your airlock. You quickly check with the starport control and confirm that you do indeed have the prefect in charge of Remnant military activities waiting at your airlock. You hit the unlock button and watch as Prefect Chilia strides onboard.`

Author: Fixed. Thanks!

			`	"Greetings, Captain <first>," he sings. "I am Prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly. "I say 'nominal' as virtually all our activities are militaristic in some way, but usually fall under the purview of those who better understand them.`
				goto main
			label chilia

> 			`The <ship> has barely come to a rest on its designated pad when your panel is blinking with a request for permission to come aboard. A quick glance at the viewscreen shows Prefect Chilia standing at your airlock. You quickly hit the accept button and watch as he strides onboard.`

Author: Fixed.

		fleet
			names "korath"
			variant
				"Korath World-Ship B (Ember)"
				"Korath Raider (Ember)" 4
	on enter "Aescolanus"
		dialog `You have located the Korath fleet. They do not seem to be going anywhere at the moment.`
	on complete
		payment 500000
		conversation
			`This time as you complete your post-flight checks you spot Prefect Chilia exiting the starling next to you and watch as he stride across the pad to your airlock. You hit the open button as he clambers up the ladder. Without a word you pull up the sensor logs from Aescolanus, highlighting the position of the various ships. His eyes flit over the display for a moment, then he nods.`
			`	"I will notify a task force. Meet me in the spaceport, and make sure your ship is ready to fight." As he departs from your ship, your commlink beeps with a message that an organization by the name of 'Remnant Logistics' has allocated you an additional <payment>.`

mission "Remnant: Cognizance 4"
	name "Destroy Korath"
	description "Prefect Chilia has asked you to eliminate the Korath fleet"
	deadline 20
	source "Caelian"
	waypoint "Aescolanus"
	to offer
		has "Remnant: Cognizance 3: done"
	on offer
		conversation

> 			`When you arrive in the spaceport you find Prefect Chilia surrounded by a group of Remnant. He is in the midst of a briefing, and waves you forward as soon as you approach. "Everyone, this is Captian <last>. They have located the Korath hunters twice now, so they will take the lead on tracking for this mission."`

Author: "amid a briefing" ? I'm not sure that sounds better than "in the midst of a briefing."

> 			`	He turns to you. "These are the captains who will be following your lead. You will have two Albatrosses, two Ibises with full complements, and 6 starlings. One of the Albatrosses is outfitted to be a shield to take the brunt of attacks." He pauses, and looks around the group. "This taskforce will be overkill for the intended target, but we are not taking chances: Make sure all the Korath die. Salvage is secondary, and there should be no escapees."`

Author: Fixed.

			`	He steps back and looks at the group. "May the Embers burn bright for you." With that, he gives the group a respectful nod and walks away. The captains all look around, make various signs you think are probably encouraging combat pilot slang, and variations on Prefect Chilia's parting words. As they disperse to their ships, one gestures to you. "Lead on, Captain."`
				accept
	npc kill
		government "Korath"
		personality staying
		system "Nenia"
		fleet
			names "korath"
			variant
				"Korath World-Ship B (Ember)"
					cargo "void sprite" 10 40
				"Korath Raider (Ember)" 4
					cargo "void sprite" 10 40
		dialog "You have destroyed all the Korath that appeared to be hunting the Void Sprites."
	npc
		government "Remnant"
		personality escort launching appeasing unconstrained disables plunders
		fleet
			names "remnant"
			variant
				"Albatross (Shield)"
				"Albatross"
				"Ibis" 2
				"Starling (Heavy)" 6
				"Petrel" 4
				"Tern" 8
	on enter "Aescolanus"
		dialog `The Korath fleet is gone. Given the urgency Prefect Chilia put on this mission, the onus is on you to find it... Quickly.`
	on complete
		conversation

> 			`The taskforce streams back into Caelian without fanfare. The ships settle onto their respective pads with a sense of satisfaction. Within seconds mechanics climbing the hull and engineers are dropping from the ceiling on tethers to inspect the engines. Batteries are plugged in, fuel lines hooked up, and carts with replacement weapnonry roll in. As always, no matter what they just did, the first responsibility of every captain is to ensure their ship is ready to head out. As the first footsteps echo on the hull of the <ship>, you switch the ship into mantenance mode before heading to the spaceport.`

Reviewer: `The task force streams back into Caelian without fanfare, Its ships settle onto their respective pads with a sense of satisfaction. Within seconds mechanics climbing the hull and engineers are dropping from the ceiling on tethers to inspect the engines. Batteries are plugged in, fuel lines hooked up, and carts with replacement weapnonry roll in. As always, no matter what they did, the first responsibility of every captain is to ensure their ship is ready to head out. As the first footsteps echo on the hull of the <ship>, you switch the ship into mantenance mode before heading to the spaceport.`

Author: Thanks, fixed!

mission "Remnant: Cognizance 5"
	name "Transport Plume to a meeting"
	description "Prefect Chilia needs Plume to consult about the void sprites."
	deadline 5
	source "Caelian"
	stopover "Aventine"
	to offer
		has "Remnant: Cognizance 4: done"
	on offer
		conversation

> 			`You find Prefect Chilia and the rest of the captains gathered in a small side room off the main cafeteria. Everyone appears to be digging into a large meal while the prefect works through the room talking with everyone. As you pull up a chair Prefect Chilia appears next to you and slides a tray in front of you. "It might not be the sort of fare you are used to from the ancestral worlds, but it is what we have to offer." He looks around at everyone eating in muted conversation. "It is something of a tradition to eat a meal together when returning from battle. Share memories of those who fall, talk about why they fell, and what we must do differently next time." He eats a few bites from his own plate.`

Reviewer 2: ```suggestion
			`You find Prefect Chilia and the rest of the captains gathered in a small side room off the main cafeteria. Everyone appears to be digging into a large meal while the prefect works through the room talking with everyone. As you pull up a chair, Prefect Chilia appears next to you and slides a tray in front of you. "It might not be the sort of fare you are used to from the ancestral worlds, but it is what we have to offer." He looks around at everyone eating in muted conversation. "It is something of a tradition to eat a meal together when returning from battle. Share memories of those who fall, talk about why they fell, and what we must do differently next time." He eats a few bites from his own plate.`
```

> 			`You find Prefect Chilia and the rest of the captains gathered in a small side room off the main cafeteria. Everyone appears to be digging into a large meal while the prefect works through the room talking with everyone. As you pull up a chair Prefect Chilia appears next to you and slides a tray in front of you. "It might not be the sort of fare you are used to from the ancestral worlds, but it is what we have to offer." He looks around at everyone eating in muted conversation. "It is something of a tradition to eat a meal together when returning from battle. Share memories of those who fall, talk about why they fell, and what we must do differently next time." He eats a few bites from his own plate.`

Reviewer: `You find Prefect Chilia and the rest of the captains gathered in a small side room off the main cafeteria. Everyone appears to be digging into a large meal while the prefect stood in front of the group, talking to everyone. As you pull up a chair, Prefect Chilia appears next to you and slides a tray in front of you. "It might not be the sort of fare you are used to from the ancestral worlds, but it is what we have to offer." He looks around at everyone eating in muted conversation. "It is something of a tradition to eat a meal together when returning from battle. Share memories of those who fall, talk about why they fell, and what we must do differently next time." He eats a few bites from his plate.`

Author: I was trying to convey the idea of the Prefect "working the room" as in talking to each captain either individually or in small pairs, one at a time.

			`	"So, what should we do differently next time?"`
			choice
				`	"A few more Albatrosses would have been great."`
				`	"The EMPs kept getting in the way."`
				`	"I think that went fairly well, actually."`

> 			`	He writes something on his notepad. "I appreciate the feedback." He pauses, and sips from his glass. "Now, moving forward, we need to know what the Korath are doing. Beyond the obvious of killing void sprites and picking fights with the Archon. The Korath are a cunning foe, and they would not antagonize the Archon for no reason." He looks thoughful.`

Reviewer: `	He writes something on his notepad. "I appreciate the feedback." He pauses, and sips from his glass. "Now, moving forward, we need to know what the Korath are doing. Beyond the obvious of killing void sprites and picking fights with the Archon, the Korath are a cunning foe, and they would not antagonize the Archon for no reason," he says.`

Author: changed.

> 			`	"I have to look into a few things here. In the meantime, please go pick up Plume from Aventine, and bring him back. We need him to be here for a briefing before <date>."`

Reviewer: `	"I have to look into a few things here. In the meantime, please go pick up Plume from Aventine and bring him back. We need him to be here for a briefing before <date>."`

Author: Fixed.

				accept
	on stopover
		dialog

> 			`Plume is wating for you at the landing pad. Once you have given the all-clear and opened the doors, he boards with little fanfare. "Thanks for dealing with the Korath, captain <last>. They have me worried." With that, he settles into a bunk and starts reviewing his notes.`

Reviewer: `Plume is waiting for you at the landing pad. Once you have given the all-clear and opened the doors, he boards with little fanfare. "Thanks for dealing with the Korath, captain <last>. They had me worried." With that, he settles into a bunk and starts reviewing his notes.`

Author: fixed.

			choice
				`	"Why do we need to go back?"`
				`	"Let's go."`
					goto action
			`	"Recordings and sensor readings are useful data, but they don't really compare to actually seeing what is going on and experiencing the data first-hand." says Plume. "There aren't too many opportunities to do it in my line of work - especially when my expertise is needed by others who would rather not spend their time watching wildlife. But I digress."`
			label action
			`	With a respectful bow to Chilia (who returns it respectfully), Plume heads for your ship. "I will be waiting onboard for you, Captain."`
				accept
	npc
		government "Korath"
		personality staying
		system "Nenia"
		fleet
			names "korath"
			variant
				"Korath Raider (Ember)" 2
	on complete
		conversation
			`The descent into the clouds of Nasqueron is becoming almost familiar to you. The whispy layers giving way to more volumous formations and the ocasional turbulent storm. Beside you, Plume is eagerly peering out the window, and you remind yourself that this may well be his first time actually seeing void spries on their world.`

> 			`	You reach the altitude that has had the most void sprites on it in the past, and activate the sensors. Plume looks around, peering out the windows. "Where are they? aren't there usually some around? They always seemed plentiful in the recordings." You focus on the sensors, and after a few minutes pick up some distant contacts. You follow them down into the lower atmosphere and eventually start seeing more void sprites."`

Reviewer: `	You reach the altitude that has had the most void sprites on it in the past before activating the sensors. Plume looks around, peering out the windows. "Where are they? Aren't there usually some around? They always seemed plentiful in the recordings." You focus on the sensors, and after a few minutes pick up some distant contacts. You follow them down into the lower atmosphere and eventually start seeing more void sprites."`

Author: Fixed.

> 			`	"That's close enough, captain. I'll go outside and watch them from the hull." You help him into the pressure suit and cycle the airlock behind him. A brief feeling of dissapointment flares within you, but you quickly suppress it. "This is, after all, Plume's job" you tell yourself as you settle back into your chair to wait for Plume to return.`

Reviewer 2: ```suggestion
			`	"That's close enough, captain. I'll go outside and watch them from the hull." You help him into the pressure suit and cycle the airlock behind him. A brief feeling of dissapointment flares within you, but you quickly suppress it. "This is, after all, Plume's job," you tell yourself as you settle back into your chair to wait for Plume to return.`
```

Reviewer: Could just be the MC talking to themselves.

Author: It *is* the MC talking to themselves. As such, looks better with the quotation marks to me.

Reviewer 2: Then I'd change the words to `mutter to yourself` rather than `tell yourself`, since telling yourself could be interpreted as thinking, not speaking aloud

Author: OK, I changed it to ` you say to yourself` so that it is clear that the player *is* talking, but it is to themselves.

Does that work for everyone?
@Anarchist2 @TheMarksman-ES

mission "Remnant: Cognizance 7"
	name "Return Plume to Caelian"
	description "Plume wants to check his records of the research on void sprites to verify some information."
	landing
	source "Nasqueron"
	destination "Aventine"
	to offer
		has "Remnant: Cognizance 6: done"
	on offer
		conversation

> 			`After several hours Plume returns to the ship, spurred by the alarms in the suit warning him about the air levels. As you help him out of the suit he looks like he is having trouble deciding if he should be thrilled or worried.`

Reviewer: `After several hours Plume returns to the ship, spurred by the alarms in the suit warning him about his air levels. As you help him out of the suit he looks like he is having trouble deciding if he should be thrilled or worried.`

Author: Fixed.

> 			`	"It is wonderful to be here, watching them from a distance. And a few did venture close, too. But I'm worried about them. Most of the void sprites out there have scorch marks similar to the scoring left by Korath weapons. And we already noticed that they are basically hiding much deeper in the atmosphere than normal." He glances down at his datapad then back up at you. "It looks like I will need to double check my records and samples back on Aventine. Once we are back into the chaos of the Ember Waste I will send Plume a signal."`

Reviewer: `	"It is wonderful to be here, watching them from a distance. A few did venture close, too. But I'm worried about them. Most of the void sprites out there have scorch marks similar to the scoring left by Korath weapons. And we already noticed that they are hiding much deeper in the atmosphere than normal." He glances down at his datapad then back up at you. "It looks like I will need to double-check my records and samples back on Aventine. Once we are back into the chaos of the Ember Waste I will send Plume a signal."`

Author: Fixed.

				accept
	npc
		government "Korath"
		personality staying
		system "Nenia"
		fleet
			names "korath"
			variant
				"Korath Raider (Ember)" 3
	on complete
		conversation

> 			`As you descend on Aventine Plume gestures towards the pad closest to his lab. "You might as well set down there. If the samples in my lab check out, I will have another mission for you right away. You will need to keep your high-pressure ship active for this one. Stop by the lab in an hour or two and we will be ready to start loading."`

Reviewer 2: ```suggestion
			`As you descend on Aventine, Plume gestures towards the pad closest to his lab. "You might as well set down there. If the samples in my lab check out, I will have another mission for you right away. You will need to keep your high-pressure ship active for this one. Stop by the lab in an hour or two and we will be ready to start loading."`
```

mission "Remnant: Cognizance 8"
	name "Transport engineer to Esquiline"
	description "An engineer working for Plume needs transportation for himself and some equipment to Esquiline."
	blocked "Plume's engineer needs a free bunk and 8 tons of cargo space for his equipment. Come back when you have enough space."
	cargo equipment 8
	passengers 1
	source "Aventine"
	destination "Esquiline"
	to offer
		has "Remnant: Cognizance 7: done"
	on offer
		log "People" "Gavriil" `A technician that works with Plume, and a talented astrobiologist. Gavriil is the project lead for the repurposed esquiline station.`
		conversation

> 			`You walk into Plume's lab to be greeted by a large stack of crates and equipment. Plume and another remnant are gathered around the stack checking items off a list. "Ah, <first>! Good timing. We were just finishing the inventory of supplies that need to go on this delivery. He gestures to the engineer, "this is Gavriil, one of our best technologists and a skilled astrobiologist in his own right. He will be going with you."`

Reviewer 2: ```suggestion
			`You walk into Plume's lab to be greeted by a large stack of crates and equipment. Plume and another remnant are gathered around the stack checking items off a list. "Ah, <first>! Good timing. We were just finishing the inventory of supplies that need to go on this delivery. He gestures to the engineer, "This is Gavriil, one of our best technologists and a skilled astrobiologist in his own right. He will be going with you."`
```

			`	"It will be a pleasure to work with you" sings Gavrill, making a short bow. "We will be traveling to a research station on Esquiline. Yes, it is a gas-world." He smiles at that. "I have to get some equipment setup and get the core systems online. There will be a few more loads of materials afterwards." He looks askance at you. "Shall we get started?"`
				accept
	on complete
		payment 100000
		conversation
			`Gavriil gives you precise directions to a co-ordinate that your sensors tell you is an empty area, and when you reach the designated spot, you see a small station shimmering out of the clouds. "We never managed to actually cloak the station," grins the engineer. "But we managed to make it hide very effectively against every form of scanning we knew of. We even managed to give it a cloak-like field that renders it mostly transparent."`
			`	He nods approvingly. "Rather impressive what my predecessors managed to do with what they had back then."`
			choice
				`	"What do you mean?"`
				`	"Let's get on with it."`
					goto skipinfo

> 			`	"Well, this place is fairly old. It was built shortly after we learned how to build Starlings, and we had trouble extending the cloaking field beyond small ships. I don't know the mechanics of it, but it has taken close to two centuries of study and development for them to get cloaks to the point where they are feasible on large ships." We had plenty of gascraft back then, and the techniques for building high-pressure systems was well understood. The Alphas hadn't shown any interest in gas giants or 'gaslined ships' as we call them, so having somewhere like this to hide in makes sense. Best place to hide is somewhere no-one will look, right?." With that he nods to himself, climbs to his feet, and heads towards the suit locker.`

Reviewer: `	"Well, this place is fairly old. It was built shortly after we learned how to build Starlings where we learned of the troubles of extending the cloaking field beyond small ships. I don't know the mechanics behind it, but it has taken close to two centuries of study and development for them to get cloaks to the point where they are feasible on large ships." We had plenty of gascrafts back then, and the techniques for building high-pressure systems were well understood. The Alphas hadn't shown any interest in gas giants or 'gaslined ships' as we call them, so having somewhere like this to hide in makes sense. Best place to hide is somewhere no-one will look, right?." With that, he nods to himself, climbs to his feet, and heads towards the suit locker.`

Author: Fixed.

			label skipinfo

> 			`	The station should be pressurized to normal atmosphere, but it has been running on standby for ages with just yearly maintnance. It is time to get this place up and running again." With two of you running the equipment, everything is unloaded in short order. You notice that the last crate includes a substantial amount of field rations and survival gear. It looks like he is planning on living rough for a while.`

Reviewer: `	"The station should be pressurized with a normal atmosphere, but it has been running on standby for ages with just yearly maintenance. It is time to get this place up and running again." With two of you running the equipment, everything is unloaded in short order. You notice that the last crate includes a substantial amount of field rations and survival gear. It looks like he is planning on living rough for a while.`

Author: Fixed.

mission "Remnant: Cognizance 9"
	name "Pick up a technician and more supplies and deliver to Esquiline."
	description "You need to pick up a technician and their equipment from Caelian and take them to Esquiline."
	blocked "Plume's technician needs a free bunk and 8 tons of cargo space for his equipment. Come back when you have enough space."
	cargo equipment 8
	passengers 1
	landing
	source "Esquiline"
	stopover "Caelian"
	to offer
		has "Remnant: Cognizance 8: done"
	on offer
		dialog
			`As you look around this earily silent station you get a message on your comlink: Plume requests that you pick up another technician and 8 tons of equipment from Caelian and bring it back to Esquiline. `
				accept
	on stopover
		dialog
			`Flight control gives you directions to a landing pad that already has a large pile of crates stacked next to it. As you land, workers swarm out to help load the equipment. Once everything is stowed, a young technician climbs aboard, stows their gear, and lets you know that everything is ready for departure. You shake your head, impressed at how quickly the Remnant make things happen when projects start to move.`
	on complete
		payment 100000
		conversation

> 			`On the flight over the technician joins you on the control deck to chat. She's a biotech specialist, adept at managing life-support systems in alien atmospheres, and a recent graduate from a Remnant educational program that sounds kind of like university. Apparently she studied astrobiology, and is part of a project aimed at supporting mutually exclusive environments in close proximity. She is eager to be taking part in this current operation, as working on the systems for a covert station are a good test for her skills.`

Reviewer: `On the flight over, the technician joins you on the control deck to chat. She's a biotech specialist, adept at managing life-support systems in alien atmospheres, and a recent graduate from a Remnant educational program that sounds kind of like university. Apparently she studied astrobiology, and is part of a project aimed at supporting mutually exclusive environments in close proximity. She is eager to be taking part in this current operation, as working on the systems for a covert station are a good test for her skills.`

Author: Fixed.

> 			`	As you approach Esquiline, it occurs to you that the Remnant might appreciate you taking steps to maintain the secrecy of the station. Altering the <ship>'s path slightly, you swing down into Esquiline in an oblique path, then meander around the planet under the cloud layer until arriving at the station. Your passenger nods approvingly as she notices the flight path. "Secrecy is always good with projects like this. We appreciate the extra effort, and if anyone were to follow you, a path like that through the clouds would leave many signs to spot."`

Reviewer: `	As you approach Esquiline, it occurs to you that the Remnant might appreciate you taking steps to maintain the secrecy of the station. Altering the <ship>'s path slightly, you swing down into Esquiline in an oblique path, then meander around the planet under the cloud layer until arriving at the station. Your passenger nods approvingly as she notices the flight path. "Secrecy is always good with projects like this. We appreciate the extra effort, and if anyone were to follow you, a path like that through the clouds would leave few signs to spot."`

Author: Fixed.

			`	The <ship> settles into the landing bay, and the technician quickly disembarks. Apparently there must have been other shipments while you were gone, as there are now a handful of people visible and quite a bit more equipment stacked everywhere. The station itself feels more alive, the air fresher. As before, you promptly receive notification of another payment of <payment> to your account.`

mission "Remnant: Cognizance 10"
	name "Remnant Data Delivery"
	description "Deliver a datacube to the engineers on Viminal."
	landing
	source "Esquiline"
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 9: done"
	on offer
		conversation
			`As you finish unloading you spot Gavriil approaching. He gestures in greeting as he approaches. "Thanks for the delivery. We've got standard logistics routes set up now for the rest."  He pauses and fishes a datacube out of a pouch. "Could you take this to Prefect Taely on Viminal? She is overseeing some retrofits and will need this data.`
				accept
	on complete
		conversation

> 			`On viminal you easily locate the maintenance yards and eventually find Taely burried in a Pelican. She spots you approaching and frees herself from the machinery. "Greetings <first>. How is our station faring?" You reply that it is improving rapidly, and hand her the datacube. "Ah, good. The detailed chemical analysis and profiles." She slots it into her console and taps a few commands. "OK, I just need to incorporate these profiles into the system and make a few more adjustments. Meet me in the cafeteria in a couple hours."`

Reviewer 2: Capitalised Viminal and changed OK to Okay
```suggestion
			`On Viminal you easily locate the maintenance yards and eventually find Taely buried in a Pelican. She spots you approaching and frees herself from the machinery. "Greetings <first>. How is our station faring?" You reply that it is improving rapidly, and hand her the datacube. "Ah, good. The detailed chemical analysis and profiles." She slots it into her console and taps a few commands. "Okay, I just need to incorporate these profiles into the system and make a few more adjustments. Meet me in the cafeteria in a couple hours."`
```

mission "Remnant: Cognizance 11"
	name "Travel with the <escort target ship> to pick up a shipment of atmosphere from Nasqueron and deliver it to <destination>."
	description "Go meet up with escorts"
	cargo equipment 8
	source "Viminal"
	stopover "Nasqueron"
	destination "Esquiline"
	npc accompany
		government "Remnant"
		personality timid launching escort
		fleet
			names "remnant"
			cargo 8 equipment
			variant
				Pelican
				Gascraft 4
	to offer
		has "Remnant: Cognizance 10: done"
	on offer
		conversation

> 			`In the cafeteria you are met by the Taely and several other captains. "Thank you for joining us," she signs. "The goal of this mission is to travel to Nasqueron together with a cargo of equipment designed to harvest some atmosphere from the void sprite's worlds, and bring it back to Esquiline. The equipment on the gascrafts will collect, compress, and store the gas, and them you can offload it to specialy prepared tanks on the pelican. Then proceed to Esquiline and offload what you have collected to the station.`

Reviewer: `In the cafeteria, you are met by Taely and several other captains. "Thank you for joining us," she signs. "The goal of this mission is to travel to Nasqueron together with a cargo of equipment designed to harvest some atmosphere from the void sprite's worlds before bringing it back to Esquiline. The equipment on the gascrafts will collect, compress, and store the gas, which you can offload to specially prepared tanks on the pelican. Then proceed to Esquiline and offload what you have collected to the station.`

Author: Changed.

				accept
	on stopover
		dialog
			`You cruise through the atmosphere at a relaxed pace, waiting while the equipment does its work. You see fewer Void Spritest than usual, many have scars. Once the equipment indicates that it has reached capacity you climb up to where the pelican is waiting in the high atmosphere, hook up the connections to offload the gas, then head back down into the depths of the planet. After several hours the captain of the pelican informs you that the tanks are full, and it is time to head to Esquiline.`
	on complete
		event "remnant: postverta reveal"
		conversation

> 			`You arrive at the hidden station on Esquiline, where the Gavriil is waiting to offload the atmosphere. His team quickly hooks up the gascraft up to ports on the bay walls and the equipment starts disgorging its contents into the system. After a few minutes they disconnect the ships, and you and the gascrafts head up to the pelican to refill your reservoirs and ferry it down to the station.`

Reviewer: `You arrive at the hidden station on Esquiline, where Gavriil is waiting to offload the atmospherice gasses. His team quickly hooks up the gascraft up to ports on the bay walls and the equipment starts ejecting its contents into the system. After a few minutes, they disconnect the ships, and you and the gascrafts head up to the pelican to refill your reservoirs and ferry it down to the station.`

Author: Fixed.

			distance 1408
			period 554
			sprite planet/station_emitter1
		object
			distance 1208
			period 654
			sprite planet/rock10
		object
			distance 439
			period 143
			sprite planet/desert7
		object
			distance 814
			period 362
			sprite planet/fog0
			object
				distance 202
				period 21
				sprite planet/dust4
		object
			period 19
			sprite star/m4
		pos 297 269
	unvisit Levana
	unvisit Caeculus
	unvisit Prosa
	unvisit Egeria
	system Levana
		add object wormhole_barren
			distance 2313
			period 389
			sprite planet/wormhole-red
	system Caeculus
		add object wormhole_barren2
			distance 2400
			period 391
			sprite planet/wormhole-red
	system Prosa
		add object wormhole_barren2
			distance 2313
			period 389
			sprite planet/wormhole-red
	system Egeria
		add object wormhole_barren
			distance 2313
			period 389
			sprite planet/wormhole-red
		

mission "Remnant: Cognizance 12"
	name "Investigate strange sensor readings"
	description "Long range sensors have picked up strange gravitational waves. Go investigate."
	landing
	source "Viminal"
	waypoint
		system Postverta
	stopover
		planet "Ssil Vida"
	to offer
		has "Remnant: Cognizance 11: done"
	on offer
		conversation

> 			`You stop by the starport on and are quickly accosted by a Remnant who informs you that Prefect Chilia is on his personal warship, and escorts you to a nearby starling. Inside you find Chilia and several other Remnant clustered around a holographic display and some display panels. "Ah, Captain. Join us, please." Chilia gestures for you to join them around the displays. As you approach, you note that the holographic display shows the Ember Waste, although there are a few details that do not match normal star maps.`

Reviewer: `You land by the starport and is immediately accosted by a Remnant who informs you that Prefect Chilia is on his personal warship before escorting you to a nearby starling. Inside you find Chilia and several other Remnants clustered around a holographic display and some display panels. "Ah, Captain. Join us, please." Chilia gestures for you to join them around the displays. As you approach, you note that the holographic display shows the Ember Waste, although there are a few details that do not match normal star maps.`

Author: Fixed. Thanks!

> 			`	Chilia notices you eyeing the deviations. "Perceptive, captain. Our sensors have reported some unusual energy signatures coming from this location, and scouts in the area say they are picking up signs of a new system. Unfortunately, they could locate neither a hyperjump route nor a wormhole that would allow them to reach it." He looks up at you. "We are hoping that you might be able to figure out a way in."`

Reviewer: `	Chilia notices you eyeing the deviations. "Perceptive, captain. Our sensors have reported some unusual energy signatures coming from this location. Scouts in the area says they are picking up signs of a new system. Unfortunately, they could neither locate neither a hyperlane nor a wormhole that would allow them to reach it." He looks up at you. "We are hoping that you might be able to figure out a way in."`

Author: Fixed.

	on stopover
		dialog
			`The station is surprisingly still active, despite its apparent age. You poke around a bit, but its mysteries remain just that.`
	on complete
		conversation
			`As you arrive back at Viminal you broadcast a warning to the port authorities that you have something following you. You inform them that it isn't hostile, but it is an unknown that originated in the new system.`

mission "Remnant: Cognizance 12b"
	landing
	invisible
	source "Ssil Vida"
	destination "Viminal"
	to offer
		has "Remnant: Cognizance 12: active"
	to complete
		has "Remnant: Cognizance 12: done"
	npc accompany
		government "Indigenous Lifeform"
		personality waiting mute
		ship "Void Sprite Elder" "Unknown Entity"
	

mission "Remnant: Cognizance 13"
	name "Follow the Creature"
	description "The creature seems to be creating constellations and trying to get you to follow it."
	source "Viminal"
	waypoint
		system Peragenor
		system Convector
		system Nenia
	stopover Nasqueron
	to offer
		has "Remnant: Cognizance 12: done"
	on offer
		conversation

> 			`Quite a few Remnant have gathered around the Starport, watching the movements of the strange creature. It gestures wildly in various ways, then freezes in several different positions with the tips of certain tentacles glowing with energy. You note that several anti-air batteries are in position around the area in case the creature becomes violent.`

Reviewer 2: Starport?

Author: Starport, as in the larger area encompassing  landing pads, outfitters, shipyards, and other facets of ship maintnance and servicing. If it was just a normal ship, I'd have said "landing pad." But this alien creature decidedly isn't following conventions and is just occupying whatever empty space it wants, irrespective of what it that space is labled. So, I'm referring to the larger area instead.

> 			`Quite a few Remnant have gathered around the Starport, watching the movements of the strange creature. It gestures wildly in various ways, then freezes in several different positions with the tips of certain tentacles glowing with energy. You note that several anti-air batteries are in position around the area in case the creature becomes violent.`

Reviewer: `Quite a number of Remnants have gathered around the Spaceport, watching the movements of the strange creature. It gestures wildly in various ways, then freezes in several different positions with the tips of certain tentacles glowing with energy. You note that several anti-air batteries are in position around the area in case the creature becomes violent.`

Author: Changed.

			`	Plume chimes in "If those freezes indicate star positions, then the flying and landing behavior in between them may be indicating it wants you to go. Will you give it a try?"`
				accept
	npc accompany
		government "Indigenous Lifeform"
		personality mute
		ship "Void Sprite Elder" "Unknown Entity"
	on enter Peragenor
		dialog `The creature comes to a stop in front of you, spins around briefly, then spreads out its tentacles in another pattern. Following the Remnant's intuition, you match it to your star charts as probably being Convector. After holding the position for a minute, the creature realigns itself and starts to shimmer with the tell-tale signs of starting to make a jump, but it holds it, as if waiting for you.`
	on enter Convector
		dialog `It seems the analysis of the location was right. The creature quickly approaches you and repeats its previous dance, ending up with a slightly different frozen position. From this vantage, you suspect that it is probably Nenia. This time, as you watch it more closely, it looks like it is following in your wake somehow.`
	on enter Nenia

> 		dialog `You have never seen such activity from the Void Sprites. Many quickly congregate around the strange ship, and seem to escort it to Nasqueron. It seems like that is where it wanted to be. In the distance, you can see the archon methodically pulverizing a korath ship, presumably another raider.`

Reviewer 2: ```suggestion
		dialog `You have never seen such activity from the Void Sprites. Many quickly congregate around the strange ship, and seem to escort it to Nasqueron. It seems like that is where it wanted to be. In the distance, you can see the Archon methodically pulverizing a Korath ship, presumably another Raider.`
```

			`	You feel a sudden pressure in your mind, and a sense of presence as the world around you fades out somewhat.`
			`	"Know you not, do I. Live creature inside dead creature, like ones who flew but not everywhere." You are momentarily disoriented as you look around and see hundreds of views of yourself peering out the cockpit window. "Your intention reverberates - Know and not know - Return with more echoes - perhaps I shall hear."`
			`	With that, the pressure vanishes and you are left staring out the window at the rapidly dispersing flock of void sprites. You aren't entirely sure what the voice meant, but you are quite sure that the Remnant are going to want to hear about this.`
	on complete
		event "remnant: nenia elder"

event "remnant: nenia elder"
	system "Nenia"
		add fleet "Void Sprites Elder" 10000
	

mission "Remnant: Cognizance 14"
	name "Go collect some void sprites"
	description "Escort the pelican fleet to Nenia and back to Arculus"
	source
		government "Remnant"
	landing
	destination Viminal
	waypoint "Nenia"
	to offer
		has "Remnant: Cognizance 13: done"
	on offer
		conversation

> 			`It is almost starting to feel commonplace to have remnant pouncing on your ship and using your cockpit as a meeting room. This time, it is Prefect Chilia and Plume who seem to appear out of nowhere requesting permision to board. The two gather around your console to watch the recording of your trip. They nod at your narration of what transpired, and focus intently when you describe the strange presence in your mind. When you finish recounting your experience, they lean back in thought.`

Reviewer 2: ```suggestion
			`It is almost starting to feel commonplace to have Remnant pouncing on your ship and using your cockpit as a meeting room. This time, it is Prefect Chilia and Plume who seem to appear out of nowhere requesting permision to board. The two gather around your console to watch the recording of your trip. They nod at your narration of what transpired, and focus intently when you describe the strange presence in your mind. When you finish recounting your experience, they lean back in thought.`
```

Reviewer: Remnants*

Author: I kind of think that "Remnant" might be one of those words that are the same for both singular and plural. A bunch of Remnant. Lots of Remnant. Lots of Remnants. Eh. Its an odd situation.

> 			`	"Captain <first>, the exact meaning is unclear, but it seems to be the voice of something not acustomed to speaking to us or creatures like us. Regardless, I think we should move forward with our plan" signs Prefect Chilia. "Best case scenario, we can deprive the Korath of easy access to the void sprites. Worst case scenario, we can move enough that - should the Korath hunt them to the extinction in Nenia, they will continue to exist in our territory."`

Reviewer: `	"Captain <first>, the exact meaning is unclear, but it seems to be the voice of something not accustomed to speaking to us or creatures like us. Regardless, I think we should move forward with our plan," signs Prefect Chilia. "Best case scenario, we can deprive the Korath of easy access to the void sprites. Worst case scenario, we can move enough that - should the Korath hunt them to the extinction in Nenia, they will continue to exist in our territory."`

Author: Fixed.

			`	Plume signals his cautious agreement. "This message is intriguing, but I do not think it should be our first priority. That being said, perhaps we should send along some additional remnant who can focus on the possibilities of more such messages."`
			`	The two look at you, and Chilia signs authoratively, "We are evacuating a large group of void sprites from Nenia to Esquiline. The atmospheres are largely a match, and we have reconfigured the station to provide the nutrients that aren't otherwise present. We have a fleet of Pelicans that have basically been hollowed out so we can lure as many void sprites into it as we can."`
			choice
				`	"How are we supposed to lure them inside?"`
				`	"Are we sure this is a good idea?"`
					goto goodidea

> 				`	"OK, let's go do this."`

Reviewer 2: ```suggestion
				`	"Okay, let's go do this."`
```

					goto letsgo
			`	Plume looks a bit embarassed. "Centuries ago when we first studied the void sprites, we learned how to harvest them. As part of that, we discovered that they were particularly interested in certain types of radiation. Likewise, if they discovered a certain kind of element common in some asteroids, they would ignore almost everything while they chewed on it. So, we have emitters inside the cargo holds, along with asteroids to keep them busy for a short while."`
			choice
				`	"Are we sure this is a good idea?"`

> 				`	"OK, let's go."`

Reviewer 2: ```suggestion
				`	"Okay, let's go."`
```

					goto letsgo
			label goodidea

> 			`	Chilia signs with a resolute tone. "Is it a good idea? Maybe, maybe not. Is it the best idea we have so far? Yes." He sighs "I fully expect the Archon to interfere: they do have long memories. As best we can tell, they can pick out who is leading a fleet, so you should expect to have them talking to you. That is, if they do not simply start shooting, anyway."`

Reviewer: `	Chilia signs with a resolute tone. "Is it a good idea? Maybe, maybe not. Is it the best idea we have so far? Yes." He sighs, "I fully expect the Archon to interfere: they do have long memories. As best we can tell, they can pick out who is leading a fleet, so you should expect to have them talking to you. That is, if they do not simply start shooting, anyway."`

Author: Fixed.

			label letsgo

> 			`	Chilia continues "You don't need to actually land anywhere in Nenia this time, Captain. Just make sure the Pelicans get there, loadup, and return safely." The two stand up and get ready to leave. "May the Embers burn bright for you, Captain." With that, they disembark and quickly clear the blast area around the landing pad. Around you a large group of pelicans are warming up, each one connecting in to your IFF and confirming their status."`

Reviewer 2: IFF?

Reviewer: Identify Friend or Foe. Although, I think Transponder is the correct word here.

Author: That is correct, IFF = Identify Friend or Foe. It is a subsystem of your targeting computer that keeps track of who the friendly people are.

Technically speaking, the Transponder is on their ship, and is the part emitting the identifying signal that your ship recognizes. (In aviation, the transponder is on the aircraft, and the receiver is on the tower. Some aircraft - notably military aircraft and some larger commercial aircraft - also have receivers. So while only some aircraft can receive those signals, they *all* have transponders for emitting the signal.)

				accept
	npc save accompany
		government "Remnant"
		personality pacifist coward launching escort
		fleet
			names "remnant"
			variant
				"Pelican (Heavy Lift)" 6
	npc
		government "Remnant"
		personality launching escort disables plunders
		fleet "Large Remnant" 3
	npc evade
		government "Korath"
		personality heroic disables plunders entering
		system "Nenia"
		fleet
			names "korath"
			variant
				"Korath World-Ship B (Ember)"
				"Korath Raider (Ember)" 3
	on enter "Nenia"
		dialog `The pelicans immediatley set to work collecting void sprites, and are soon ready to go.`
	on enter "Arculus"

> 		dialog `You watch vigilantly as the pelicans with the void sprites dip low into Esquilion's atmosphere and open their bay doors. The void sprites emerge tentatively, then confidently launch themselves into the clouds. A few minutes later you receive a message from Esquiline station informing you that the first void sprite just swooped through the bay, grabbed one of the asteroid supplmements, and vanished into the clouds again.`

Reviewer 2: ```suggestion
		dialog `You watch vigilantly as the Pelicans with the void sprites dip low into Esquilion's atmosphere and open their bay doors. The void sprites emerge tentatively, then confidently launch themselves into the clouds. A few minutes later you receive a message from Esquiline station informing you that the first void sprite just swooped through the bay, grabbed one of the asteroid supplements, and vanished into the clouds again.`
```

			`	"That's close enough, captain. I'll go outside and watch them from the hull." You help him into the pressure suit and cycle the airlock behind him. A brief feeling of dissapointment flares within you, but you quickly suppress it. "This is, after all, Plume's job" you tell yourself as you settle back into your chair to wait for Plume to return.`

mission "Remnant: Cognizance 7"
	name "Return Plume to Caelian"
	description "Plume wants to check his records of the research on void sprites to verify some information."
	landing
	source "Nasqueron"
	destination "Aventine"
	to offer
		has "Remnant: Cognizance 6: done"
	on offer
		conversation
			`After several hours Plume returns to the ship, spurred by the alarms in the suit warning him about his air levels. As you help him out of the suit he looks like he is having trouble deciding if he should be thrilled or worried.`

> 			`	"It is wonderful to be here, watching them from a distance. A few did venture close, too. But I'm worried about them. Most of the void sprites out there have scorch marks similar to the scoring left by Korath weapons. And we already noticed that they are hiding much deeper in the atmosphere than normal." He glances down at his datapad then back up at you. "It looks like I will need to double-check my records and samples back on Aventine. Once we are back into the chaos of the Ember Waste I will send Plume a signal."`

Reviewer 3: `Once we are back into the chaos of the Ember Waste I will send Plume a signal."` --  wrong name?

Author: Good catch. Fixed.
